notebooks/002-openvino-api/002-openvino-api.ipynb

```diff
@@ -129,35 +129,38 @@
     "from openvino.runtime import Core\n",
     "\n",
     "ie = Core()\n",
-    "onnx_model_path = \"model/segmentation.onnx\"\n",
-    "model_onnx = ie.read_model(model=onnx_model_path)\n",
-    "compiled_model_onnx = ie.compile_model(model=model_onnx, device_name=\"CPU\")"
+    "segmentation_model_onnx = \"model/segmentation.onnx\"\n",
+    "\n",
+    "model = ie.read_model(model=segmentation_model_onnx)\n",
+    "compiled_model = ie.compile_model(model=model, device_name=\"CPU\")"
```

Comment:
This should be restored to the previous state

Reply:
Reverted to previous state